src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/abbreviationaswordinname/InputAbbreviationAsWordInNameType7.java

```diff
@@ -0,0 +1,45 @@
+/*
+AbbreviationAsWordInName
+allowedAbbreviations = ORDER, OBSERVATION, UNDERSCORE, TEST
+
+
+*/
+
+
+package com.puppycrawl.tools.checkstyle.checks.naming.abbreviationaswordinname;
+
+public class InputAbbreviationAsWordInNameType7 {
+    // violation below 'Abb.* in name '.*' must contain no more than '4' consecutive cap.* let.*.'
+    private int HBCK_LOCK_PATH;
+
+    // violation below 'Abb.* in name '.*' must contain no more than '4' consecutive cap.* let.*.'
+    Boolean[] BOOL_VALS = { false, true };
+
+    void getTEST() {
+    } // ok
+
+    void getORDER_OBSERVATION() {} // ok
+
+    // violation below 'Abb.* in name '.*' must contain no more than '4' consecutive cap.* let.*.'
+    void getNONE_Test() {}
```

Comment:
Make violation message like:
`name 'NONE' must contain no more than '4' .*cap`

We need to see name.